lib/checkPullRequestJob.js

```diff
@@ -32,6 +32,101 @@ const JOB_REGEX = new RegExp(
   ].join('')
 );
 
+const CRON_REGISTRY_FILENAME = ['cron.yaml', 'main_cron.py', 'cron.py'];
+const CRON_JOB_REGEX = new RegExp(
+  [
+    '(?<addition>\\+)(?<classDefinition>class\\s)',
+    '(?<name>[a-zA-Z]{2,256})(?<suffix>Handler)(?<funDef>\\()'
+  ].join('')
+);
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem} file
+ */
+const getNewCronJobsFromFile = (file) => {
+  return getNewItemsFromFileByRegex(CRON_JOB_REGEX, file);
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem} file
+ */
+const addsNewCronJob = (file) => {
+  const newJobs = getNewCronJobsFromFile(file);
+  return newJobs.length > 0;
+};
+
+const compare = (CRON_REGISTRY_FILENAME, changedFiles) => {
+  CRON_REGISTRY_FILENAME.forEach((e1) => changedFiles.forEach((e2) => {
+    if(e1 === e2) {
+      return true;
+    }
+  }));
+}
```

Comment:
I'm not sure I understand what this function is doing.

Reply:
@jameesjohn I am trying to compare the values of the two arrays .If `changedFiles` contain any file which is also in `CRON_REGISTRY_FILENAME` then `getCronCommentBody ` should work